department-of-veterans-affairs/va.gov-cms
# VACMS-7649 more claro mob session fixes

## Description

Relates to #7649. Fixes another grab bag of issues surfaced from the mob session.

- fixes the a11y error reported in vagovclaro's right rail
- fixes linkit contrast issue & scrollbar for long lists of results
- text format help icon returns
- checkbox helptext placement corrected
- breadcrumb contrast updated to pass a11y
- consistent form error message styles
- view tab columns standardized
- site alerts get correct styles (pinned to bottom of page, site shield covers full page)
- focus state spacing adjusted for text inputs
- alert classes ordered for precedence in cascade if multiple classes applied (error beats warning beats status)
- service location spacing fixes, address no longer breaks layout

## Testing done
locally, verified all of the above between local env + claro control & demo tugboat envs

## Screenshots
linkit contrast + scrollbar (not pictured)
![Screen Shot 2022-01-21 at 1 44 08 PM](https://user-images.githubusercontent.com/11279744/150892508-82a51498-6af7-44c8-9657-7c3fadb9477f.png)
text format help icon
![Screen Shot 2022-01-21 at 2 06 39 PM](https://user-images.githubusercontent.com/11279744/150892512-0c2cd242-3a44-4bdf-8d3a-14daa78a3b00.png)
right rail help text color, passes a11y
![Screen Shot 2022-01-21 at 2 06 50 PM](https://user-images.githubusercontent.com/11279744/150892515-4fdda388-6449-458e-a69f-74038c3fb8bd.png)
error message consistent with other fields
![Screen Shot 2022-01-21 at 4 18 57 PM](https://user-images.githubusercontent.com/11279744/150892516-11b8dc3b-e8ca-4352-8cba-e03a7b2a7a5c.png)
breadcrumb contrast
![Screen Shot 2022-01-24 at 5 17 18 PM](https://user-images.githubusercontent.com/11279744/150892628-e6c760f3-b981-4cf3-b532-09659665020e.png)
focus state spacing, colors updated
![Screen Shot 2022-01-24 at 5 17 39 PM](https://user-images.githubusercontent.com/11279744/150892631-be4bf945-c39a-41b7-8813-69e04f7eccb4.png)

checkbox color for hover/focus state updated
![Screen Shot 2022-01-24 at 5 20 03 PM](https://user-images.githubusercontent.com/11279744/150892860-efecaff8-9797-49f6-b626-a458890ef114.png)
![Screen Shot 2022-01-24 at 5 20 06 PM](https://user-images.githubusercontent.com/11279744/150892862-3115ac0e-8876-494d-9eb0-2ceb9a2461d1.png)

service location spacing updated, address field no longer breaks out of layout on smaller screen sizes
![Screen Shot 2022-01-24 at 5 23 09 PM](https://user-images.githubusercontent.com/11279744/150893222-cb4e2fa8-0a7f-479c-9ed6-1a2b66593ea0.png)

## QA steps

these nodes:
/node/42150/edit (service location)
/node/6108/edit (checkboxes in form section)
/node/67/edit (focus state spacing on text inputs, right rail, error messages), view tab to see spacing update
/node/add/page (text format help icon, linkit fixes (add link inside rich text field))

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
